### ISSUE TITLE
V 0.41.0.0. crashes and deletes reports

### ISSUE THREAD

**Issue by gsurrel** [mozilla/MozStumbler#1103]
Since I've updated to 0.41.0.0, MozStumbler seems to crash often as the service is not running anymore after my usual 15 minutes of cycling. I didn't encounter this problem for the previous pre-releases.

I tried to see what's going on in the log, but the _only_ thing I have is "HTTP non-success code: 400, 400 Error, deleting bad report". Is it possible to see what is the actual report so as to check if it's really wrong or if it's just a network problem (captive login portal of something like that) ?

(I've checked the "Enable crash reporting" feature in the settings, I hope it helps)

**Comment by Djfe** [comment]
@crankycoder a question about the crash reporter:
will it report old crashes too if it gets turned on or only the new ones?

@gsurrel if your battery percentage was above 15% all the time then the cause was really a crash but I cannot say more about it

**Comment by gsurrel** [comment]
The crash reporter was enabled before the crash happened, so there is nothing to worry about, concerning this specific point :)
Concerning the battery, it was always between 80% and 100%

**Comment by garvankeeley** [comment]
Weird, we have zero crash reports for 0.41.0 so far, unless @crankycoder has seen one that I missed.

**Comment by narfg** [comment]
@garvankeeley I use version 0.41.0.0 and it crashed a couple of times. With adb logcat I can see that ACRA sends crash reports.

**Comment by gsurrel** [comment]
Is there a way that the MozStumbler service quits without crashing while it's scanning, apart from battery going low?
EDIT: On my way back home, it was still running at mid-way, but once I was home, it was off. Could it be it's automatically turning off when it reconnects to a known wifi?

**Comment by gsurrel** [comment]
I've not noticed this problem since then. I don't know if it's because it's hard to reproduce or because of v42. I think this issue can be closed by the end of the week if it's stays like that

**Comment by gpoo** [comment]
Yesterday I noticed a similar behavior in an Nexus S / Andriod 4.0.4, and I recalled this issue.  (Although, it was with 0.40).  It never was under 15% of battery; and the user told me that from time to time it was necessary to run MozStumbler, and rather than bringing the application, it showed a crash dialog, and no reports, no nothing.  After upgrading to 0.42, the summary of reports sent appeared again. Not sure if some reports were lost in the meantime.  The problem did not appear again after the upgrade.
